linux-tkg-config/prepare

```diff
@@ -1757,6 +1757,16 @@ CONFIG_DEBUG_INFO_BTF_MODULES=y\r
   msg2 "Setting config"
   make ${_config_updating} ${llvm_opt} |& tee -a "$_where"/logs/prepare.log.txt
 
+  # Modify the config file to fit Fedora SELinux configuration
+  if [ "$_distro" = "Fedora" ]; then
+    msg2 "SELinux activation for Fedora"
+    sed -i 's/# CONFIG_AUDIT is not set/CONFIG_AUDIT=y/' .config
+    sed -i 's/# CONFIG_SECURITY_SELINUX is not set/CONFIG_SECURITY_SELINUX=y/' .config
+    sed -i 's/# CONFIG_DEFAULT_SECURITY_SELINUX is not set/CONFIG_DEFAULT_SECURITY_SELINUX=y/' .config
+    sed -i 's/CONFIG_DEFAULT_SECURITY_DAC=y/# CONFIG_DEFAULT_SECURITY_DAC is not set/' .config
+    sed -i 's/.*CONFIG_LSM=".*/CONFIG_LSM="lockdown,yama,integrity,selinux,bpf,landlock"/' .config
```

Comment:
You can define this function along with `_enable` and co
```bash
_set_str() {
  scripts/config -k --set-str "$1" "$2"
}
```
then use it instead of `sed`.